source/extensions/filters/network/thrift_proxy/thrift.h

```diff
@@ -22,6 +24,7 @@ enum class TransportType {
 
 /**
  * Names of available Transport implementations.
+ * TODO(kuochunghsu): rename class name.
```

Comment:
my apologies. this is note to myself so i'll address it for your second pass.

Reply:
I end up introduce `ProtoUtils` so `ProtoUtils::getTransportType(thrift_config.transport())` could
do the query. Therefore `TransportNames::fromType` is not extented and don't need renamed.
Also use `switch` to avoid additional fiasco memory.